src/components/20-molecules/datepicker/utils/date.js

```diff
@@ -1,7 +1,9 @@
 export const getStartOfWeek = date => {
   const iDayOfWeek = date.getDay();
   const iDifference = date.getDate() - iDayOfWeek + (iDayOfWeek === 0 ? -6 : 1);
-  return new Date(date.setDate(iDifference));
+  const newDate = new Date(date);
+  newDate.setDate(iDifference);
```

Comment:
What does the `i` stand for?

Reply:
I suspect index or integer, but it's code that I did not touch.

I changed it to `dayOfWeek` and `daysSinceBeginningOfMonth`, thanks for the feedback!